### ISSUE TITLE
/> inside string content causes Syntax("Unexpected token: />") error

### ISSUE THREAD

**Issue by Mingun** [netvl/xml-rs#197]
Parsing such perfectly well-formed XML document (using serde-xml-rs, which depends on xml-rs) lead to error:
```rust
#[test]
fn valid_xml() {
  let _: String = serde_xml_rs::from_str("<tag>/></tag>").unwrap();
}
```
XML documents with such symbols can be, for example, produced by mature XmlBeans Java library, when you serialize XML document, embedded in another XML document as string.

_originally reported at https://github.com/RReverser/serde-xml-rs/issues/134_

**Comment by dalegaard** [comment]
This is fixed in PR #195 which is still open as of writing this.

**Comment by Mingun** [comment]
#195 was merged, so close.
